src/test/end-to-end/pages/stepsToMakingYourClaim/stepsToMakingYourClaim.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+const contactUs = require('../../helpers/contactUs.js');
+
+module.exports = async function () {
+  // eslint-disable-next-line @typescript-eslint/no-this-alias
```

Comment:
Do we need this function?

Reply:
yes this page would be tested as part of the claim Details and would be usefull for future end-2-end tests as well...